docs/references/architecture/adr-119-dog-mempool-gossip.md

```diff
@@ -0,0 +1,363 @@
+have not observed a reduction in redundancy due to a lower interval. Most likely due to the fact that, regardless
+of the interval, it takes a certain amount of time for the information to propagate through the network.  
+
+- `mempool.target_redundancy: float`: Set to `1`. The redundancy level that the gossip protocol should aim to
+achieve. An acceptable value, based on our tests was also `0,5`. While still having, for each unique transaction, one duplicate, the number of transaction messages in the system was significantly lower and we did not see a great benefit in using `0.5` over `1`. Therefore we set the default to a safer value in terms of resilience to attacks.
+
+ Overall increasing this value above `2` would lead to too many duplicates without an improvement in the tolerance of the system to byzantine attacks.  
+
+The third parameter defines the bounds of acceptable redundancy levels: 
+- `TargetRedundancyDeltaPercent: float`: Set to `0.1` (10%). It defines the bounds
+of acceptable redundancy levels. The actual redundancy will be: `redundancy +- redundancy*delta`. A lower delta 
+did not lead to a visible reduction in redundancy, while slightly increasing the number of control messages sent. 
+We have therefore opted to not reduce this value further. It does not impact the protocol as much as the target redundancy or the redundancy adjustment interval. 
+
+Part of the work on the protocol was extensive testing of its performance and impact on the network, as well 
+as the impact of the network configuration and load on the protocol itself. 
+Issue [\#4320] covers the experiments performed on DOG.
+
+We have also tested scenarios where we send each transaction to 100 of the 200 nodes. Typically we only send each transaction to one node. DOG was able 
+to reduce the number of duplicates in the system, even under a high load, even when the target redundancy is `1`. 
+
+After extensive testing, we did not find compelling evidence and differences in a variety of runs to warrant the
+tuning of the default values chosen. We have left the target redundancy and redundancy adjustment interval 
+as something operators can adjust so that users can experiment in their testnets. But none of our experiments
+found a degradation in performance and correctness for these value. 
+
+More details on the experiments used to make this decision
+ can be found in Issue [\#4320].
+
+
+
+## User recommendation
+
+- The Entire network should use DOG. Otherwise the impact will be minimal. 
+
+
+- The protocol implicitly favors faster routes, by cutting routes through peers that send the duplicate transaction at a later time.
+
+
+- If the frequency of `HaveTx` messages is too high, nodes will have too many routes cut.
+
+-  While we did not observe any single node missing out on transactions when the lowering the target 
+reduncancy event to `0.1`, we do not advise users to do this without a strong justification and testing. 
```

Comment:
We have not tested a 0.1 target with perturbations and sending each tx to multiple nodes. So no, it's not recommended.